src/viam/examples/dial/example_dial.cpp

```diff
@@ -55,7 +54,7 @@ int main() {
     }
 
     // ensure we can send requests for specific resources
-    std::vector<Name> just_one = {resource_names->at(0)};
+    std::vector<Name> just_one = {resource_names[0]};
```

Comment:
Just FYI, but there is an important difference between `std::vector::at` and `std::vector::operator[]`: the former is bounds checked but the latter is not. I'm not sure if the old code intended to call the bounds checked version or was just working around the awkwardness of `(*resource_names)[0]`.

Reply:
I don't think there was a strong desire for the bounds checking particularly, especially in example code. This looks fine to me.